commands.json

```diff
@@ -2133,7 +2134,7 @@
     "group": "list"
   },
   "RPUSHX": {
-    "summary": "Append a value to a list, only if the list exists",
+    "summary": "Append one or multiple values to a list, only if the list exists",
     "complexity": "O(1)",
```

Comment:
Should be "O(N) where N is the number of elements to be added by the operation"

Reply:
updated complexity and history in commit 9a6afdd